.changeset/three-nails-press.md

```diff
@@ -0,0 +1,8 @@
+---
+"@smithy/fetch-http-handler": minor
+"@smithy/node-http-handler": minor
+"@smithy/protocol-http": minor
```

Comment:
I think this needs a major version bump due to breaking changes in interfaces, e.g. generics.